karma.conf.js

```diff
@@ -19,7 +19,6 @@ module.exports = function(karma) {
 		// These settings deal with browser disconnects. We had seen test flakiness from Firefox
 		// [Firefox 56.0.0 (Linux 0.0.0)]: Disconnected (1 times), because no message in 10000 ms.
 		// https://github.com/jasmine/jasmine/issues/1327#issuecomment-332939551
-		browserNoActivityTimeout: 60000,
```

Comment:
I removed this line that I had added in an earlier PR because it's very annoying to use with `gulp test --watch`. It closes the browser after 10 min which makes debugging really hard. Hopefully `browserDisconnectTolerance: 3` will be enough. We could make it `browserDisconnectTolerance: 5` or something if we want to add a little extra buffer since we're removing `browserNoActivityTimeout`